docs/modules/ROOT/pages/implementations/influx.adoc

```diff
@@ -120,22 +122,18 @@ management.metrics.export.influx:
 
 Telegraf is a StatsD agent that expects a modified flavor of the StatsD line protocol.
 
-The following example shows the Gradle configuration:
+==== Gradle
 
-include::../_gradle_bom.adoc[]
-
-Add the following dependency:
+After the BOM is xref:../installing.adoc[configured], add the following dependency:
 
 [source,groovy]
 ----
 implementation 'io.micrometer:micrometer-registry-statsd'
 ----
 
-The following example shows the Maven configuration:
```

Comment:
"NOTE: The version is not needed for this dependency since it is defined by the BOM." seems to be missing.

Reply:
Thank you, fixed.